resources/images/exif/exif.go

```diff
@@ -227,7 +227,7 @@ func (e *exifWalker) Walk(f _exif.FieldName, tag *tiff.Tag) error {
 func nullString(in []byte) string {
 	var rv bytes.Buffer
 	for _, b := range in {
-		if unicode.IsPrint(rune(b)) {
+		if unicode.IsGraphic(rune(b)) {
```

Comment:
OK, taking a second look at this, there is one more error in this, as this will not handle multi-byte runes. See the loop in this example:

https://golang.org/pkg/unicode/utf8/#DecodeRune

Reply:
Commit pushed.  Fixing this should remove the need for the `utf8.ValidString` check.  PTAL